### PR TITLE
fix: expansion with re-export. but rely on expansion order.

### PR DESCRIPTION
**Pull request by TOETOE55**
Introduce an "accumulator" attribute at the end of attributes list(make it expends at the last time) to collect cfgs.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -94,50 +116,115 @@ fn cfg_vis_impl(
...
-                    "`cfg_vis` can only apply on item with visibility",
-                ));
-            }
-        };
+    let __cfg_vis_accumulator_declare_path = __cfg_vis_accumulator_declare_path();
+
+    let (_, attrs) = proj_item(&mut item);
+    let mut accumulator: Vec<_> = attrs
+        .into_iter()
+        .filter(|attr| attr.path == __cfg_vis_accumulator_declare_path)
```
Comment (mu001999) [inline src/lib.rs]: Could direct `find` be used here?

Comment (TOETOE55) [reply src/lib.rs]: I want a compile error when there are more than one accumulators.

File: src/lib.rs [hunk 2]
```diff
@@ -94,50 +127,208 @@ fn cfg_vis_impl(
...
+        .filter_map(|acc_attr| {
+            if acc_attr.path == __cfg_vis_accumulator_declare_path {
+                Some((acc_attr.parse_args(), acc_attr))
+            } else {
+                None
+            }
+        })
+        .map(|(acc, attr)| {
+            let acc: CfgVisAttrArgsAccumulator = acc?;
+            if acc.version == version {
```
Comment (mu001999) [inline src/lib.rs:145]: Does __cfg_vis_accumulator itself need to perform such a check? E.g., only one cfg_vis and __cfg_vis_accumulator but with different versions.

Comment (TOETOE55) [reply src/lib.rs:145]: You are right

**Review by mu001999 — APPROVED** [review]

**Pull request merged**